styles/useDarkMode.js

```diff
@@ -0,0 +1,21 @@
+import { useState, useEffect } from "react";
+
+export const useDarkMode = () => {
```

Comment:
`useTheme` or something like that will be more proper name. in `useDarkMode` we have `light` and `dark` mode. Not just only dark mode.